### DISCUSSION TITLE
2.4 Non-Elastic Clustering?

### DISCUSSION THREAD

**Discussion by EddieN17** [Security-Onion-Solutions/securityonion#10640] [2.4]
Hello,

I'm testing out the 2.4.2 beta with a distributed deployment and it appears that it's running using Elastic Clustering. Will there be an option to use non-elastic clustering, or with the new focus on elastic agents and fleet, will elastic clustering be the only option? I know with 2.3 there was the option during setup, but I didn't see that option with 2.4.2 using the iso.

**Comment by dougburks** [comment]
2.4 switches to Elastic clustering.
